src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -66,9 +66,11 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord, SdkLimitO
 
             try
             {
+                var timestamp = (ulong)logRecord.Timestamp.ToUnixTimeNanoseconds();
                 otlpLogRecord = new OtlpLogs.LogRecord
                 {
-                    TimeUnixNano = (ulong)logRecord.Timestamp.ToUnixTimeNanoseconds(),
+                    TimeUnixNano = timestamp,
+                    ObservedTimeUnixNano = timestamp,
```

Comment:
@CodeBlanch Just a TODO... we should decide to add a separate observed timestamp or not to our `LogRecord`. I'm leaning towards no, but technically it's part of the data model. I can't imagine a scenario for our purposes where it would not equal the regular timestamp.

Reply:
Ya I felt the same way (that we didn't need it) which is why it isn't on [LogRecordData](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api/Logs/LogRecordData.cs) at the moment. Thought maybe it is it more of a collector thing? 🤷

Reply:
Yes.. (I think this is similar to the Metric aggregation "Summary". It not needed in the SDks.)